### DISCUSSION TITLE
Components resize event

### DISCUSSION THREAD

**Discussion by ahmafi** [GrapesJS/grapesjs#4613] [Q&A]
Are there any events for components resizing? I have tried `editor.on('component:styleUpdate')`, `editor.on('component:styleUpdate:width')`, `model.on('change:style')` but none of trigger anything.

Update:

I have found `editor.on('component:resize')` but it doesn't help because it triggers on all components, and it doesn't return current component in its callback.

**Comment by rajevar** [comment]
+1 for this question.  We are currently using `editor.on('style:property:update'` but its getting triggered for every property change which doesn't seem right. 

Please see the jsfiddle for `component:styleUpdate` and `component:styleUpdate:width` issue if anyone likes to take a look.
https://jsfiddle.net/rajevar/rjmz87sa/14/
